pkgs/by-name/kc/kcl-cli/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kcl-cli";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = "cli";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ZjEMgQukhBGY3LWqsGmLj3lKfLtNNaQugQs0cSLMb80=";
+  };
+
+  vendorHash = "sha256-jmqKMB85HxAlwH7FVjHrLCZQYuAJrguRfzIz1yMypjw=";
+
+  ldflags = [
+    "-X=kcl-lang.io/cli/pkg/version.version=${finalAttrs.version}"
+  ];
+
+  subPackages = [ "cmd/kcl" ];
+
+  meta = with lib; {
+    description = "A command line interface for KCL programming language";
+    homepage = "https://github.com/kcl-lang/cli";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ peefy ];
+    mainProgram = "cli";
+  };
+})
```

```suggestion
buildGoModule rec {
  pname = "kcl-cli";
  version = "0.8.5";

  src = fetchFromGitHub {
    owner = "kcl-lang";
    repo = "cli";
    rev = "v${version}";
    hash = "sha256-ZjEMgQukhBGY3LWqsGmLj3lKfLtNNaQugQs0cSLMb80=";
  };

  vendorHash = "sha256-jmqKMB85HxAlwH7FVjHrLCZQYuAJrguRfzIz1yMypjw=";

  ldflags = [
    "-X=kcl-lang.io/cli/pkg/version.version=${version}"
  ];

  subPackages = [ "cmd/kcl" ];

  meta = with lib; {
    description = "A command line interface for KCL programming language";
    homepage = "https://github.com/kcl-lang/cli";
    license = licenses.asl20;
    maintainers = with maintainers; [ peefy ];
    mainProgram = "kcl";
  };
}
```

Just had a look at the evaluation failure and noticed that buildGoModule can't use `finalAttrs`. Apologies. However, with the suggested patch, I was able to successfully run the KCL CLI.

---

Thank you! But CI failed again.